packages/systemd/systemd.spec

```diff
@@ -118,11 +118,13 @@ CONFIGURE_OPTS=(
  -Drepart=true
  -Dcoredump=false
  -Dpstore=true
+ -Doomd=false
```

Comment:
[systemd-ood](https://www.man7.org/linux/man-pages/man8/systemd-oomd.service.8.html) seems like a nice features I wonder if we could use it to set some limits in the units we maintain when we move to cgroups v2 :thinking:

Reply:
At one point systemd-oomd required swap, which we don't support today. It's also not obvious how useful it is with kubelet in the picture.